charts/cortx/templates/control/deployment.yaml

```diff
@@ -25,8 +25,11 @@ spec:
           configMap:
             name: {{ include "cortx.configmapName" . }}
         - name: cortx-ssl-cert
-          configMap:
-            name: {{ include "cortx.tls.configmapName" . }}
+          secret:
+            secretName: {{ .Values.control.certificateSecret }}
```

Comment:
Instead, rename template `cortx.tls.configmapName` to `cortx.tls.secretName`, and derive it based on either the release name, or use the externally provided name.